msteams-platform/m365-apps/extend-m365-meeting-app.md

```diff
@@ -0,0 +1,198 @@
+
+    :::image type="content" source="images/toolkit-zip-teams-app-package.png" alt-text="Screenshot shows the Zip Teams App Package option in Teams Toolkit extension for Visual Studio Code.":::
+
+1. The preview for Teams meeting app support in Outlook requires you to sideload your app package using TeamsFx CLI. (Sideloading from the *Manage your apps* option in Teams client is not currently supported.) Ensure TeamsFx CLI is installed:
+
+    ```bash
+    npm install -g @microsoft/teamsfx-cli
+    ```
+
+1. Use the following command to sideload your app package to Teams, replacing *<path\to\appPackage.zip>* with the path to your app package.
+
+    ```bash
+    teamsfx m365 sideloading --file-path <path\to\appPackage.zip>
+    ```
+
+After it's sideloaded to Teams, your meeting app will be available for use in both Teams and Outlook. It may take several minutes and a restart of the Outlook for Windows desktop client for your app to appear in the **Apps** menu.
+
+## Preview your meeting app in Outlook
+
+Here's how to preview your meeting app running on Outlook for Windows desktop:
+
+1. With your meeting app running (either remotely from Azure, or locally from Teams Toolkit `F5`), open Outlook for Windows desktop signed in to your test tenant account. If Outlook is already running, close and restart it again.
+
+1. Open *Calendar*, then select **New Meeting**. By default, new meetings are *Microsoft Teams* meetings. For a meeting app to run in Outlook, the meeting must be a Teams meeting (meeting apps are not supported in non-Teams meetings scheduled from Outlook).
+
+1. From the meeting scheduling window, select **All Apps**. Your sideloaded meeting app should appear among your installed meeting apps.
+:::image type="content" source="images/outlook-meeting-all-apps.png" alt-text="Sideloaded meeting app showing in `All Apps` menu of Outlook meeting scheduler":::
+
+1. Select your meeting app. This opens its configuration dialog, where your user will configure any relevant app settings for their specific meeting.
+:::image type="content" source="images/outlook-meeting-app-configuration.png" alt-text="Meeting app configuration page showing from Outlook meeting scheduler":::
+
+1. Click **Add** to dismiss the dialog. Your meeting app will display in the side panel of the scheduling window and the **Apps** field will indicate its availability to participants in the meeting. From the **Apps* list, you can open the dropdown menu on your app to see how the app appears in the Store (About), update the app configuration for this meeting (Settings), or remove the app from this meeting (Remove).
```

Comment:
Please follow the writing guidelines and break it into multiple steps as too many actions are in one step.